JS101/fundamentals/rock-paper-scissors-project.md

```diff
@@ -130,6 +130,7 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [Jacob Folley's solution](https://github.com/jacob-folley/rock_paper_scissor) - [View in browser](https://jacob-folley.github.io/rock_paper_scissor/)
 - [Ivan Martinez' Solution](https://github.com/ivanmaru86/rock-paper-scissors)
 - [Jimmy's solution](https://github.com/pty5io/Rock-Paper-Scissors) - [View in browser](https://pty5io.github.io/Rock-Paper-Scissors/)
+
```

Comment:
Looks like you accidently added a newline here @MikeSS281986. Can you remove it please? 🙂 

it will break the formatting of the list if it stays

Reply:
I believe it looks correct how do I submit my change? I do not see a merge option. I'm sorry. Still new to this.

Reply:
I made a commit. Was that it? Thought you were supposed to make a merge request once I commit.